romeo/src/stacks_client.rs

```diff
@@ -163,7 +163,7 @@ impl StacksClient {
             .json()
             .await?;
 
-        Ok(fee_rate * tx_len)
+        Ok(fee_rate * tx_len * 100)
```

Comment:
Where does this 100 come from and can we store it in a constant so that is has a meaningful variable name?

Reply:
It's still unclear what is the right number for this. We've observed some transactions being stuck in the mempool for days and have suspected it might have something to do with the fees. On the other hand testnet nodes all return `1` as the fee rate constantly. At the moment this is just a random multiplier but as we find out more about testnet fees we can make sure to tidy it up.

I've created this issue https://github.com/stacks-network/sbtc/issues/98.